d2bs/kolbot/libs/bots/Enchant.js

```diff
@@ -493,6 +493,10 @@ MainLoop:
 	this.floodCheck = function (command) {
 		var cmd = command[0],
 			nick = command[1];
+			
+		if (!nick) {	// ignore overhead messages
+				return true;
```

Comment:
please fix the indentation here, other than that LGTM

Reply:
Fixed, and thanks for your time :)
Now I have to press "Close pull request" button?